dotCMS/src/main/java/com/dotcms/util/PaginationUtil.java

```diff
@@ -224,6 +224,8 @@ private static String getUrl(final String urlBase, final String filter, final in
 
 		if (page != -1){
 			params.put(PAGE, String.valueOf(page));
+		}else{
+			params.put(PAGE, "pageValue");
```

Comment:
**"pageValue"** is the default value if **page** is not set?, is not a weird default for a param?, if nothing was set we should not send the param in the first place or at least send something like **-1**
**Note:** I'm assuming the **page** param is the page number